changelogs/unreleased/gh-6983-fix-command-in-help-messages

```diff
@@ -0,0 +1,3 @@
+## bugfix/tarantoolctl
```

Comment:
Please, add `.md` to the end of the file name. Otherwise it will be ignored by the `tools/gen-release-notes` script.

Reply:
fixed